JotunnLib/Managers/PieceManager.cs

```diff
@@ -91,12 +85,18 @@ public bool AddPieceTable(CustomPieceTable customPieceTable)
                 return false;
             }
 
-            // Set Container as parent to add the prefab to the DontDestroyOnLoad scene
-            customPieceTable.PieceTablePrefab.transform.parent = PieceTableContainer.transform;
-
             // Add the prefab to the PrefabManager
             PrefabManager.Instance.AddPrefab(customPieceTable.PieceTablePrefab);
 
+            // Create all custom categories on that table
+            if (customPieceTable.Categories.Length > 0)
```

Comment:
Unneeded condition, foreach is only looped when there is anything in the categories array